src/Components/CreateImageWizard/CreateImageWizard.scss

```diff
@@ -88,7 +88,4 @@ div.pf-v5-c-alert.pf-m-inline.pf-m-plain.pf-m-warning {
     display: flex
 }
 
-// Fixes min height issues on the main wizard container
-.pf-v5-c-wizard__outer-wrap {
-    min-height: 602px;
-}
+
```

Comment:
I spent a little time looking into this and it’s pretty tricky, you have to really understand flexbox behavior.

There are several nested flexboxes on the page, and we do need them as we want the wizard to grow or shrink with the viewport so the footer is always visible.

The actual element that is growing is this one: `<div class=”pf-v5-c-wizard__main”>`. It has `flex: 1 1 auto`. These values represent `flex-grow`, `flex-shrink`, and `flex-basis`, respectively. This flexbox is able to grow and is what is causing the wizard (its parent element) to grow and cause the entire page to become scrollable.

If we change the `flex-basis` from `1` to `0`, the element will no longer be able to grow and instead the content is rendered within a scrollable element within the wizard.
 
Although both this and setting a height work, I think this might make our intentions a little clearer for future readers of the code than setting a height, since it directly addresses the cause of the growth – the flexbox behavior.

To sum up, I think adding
```
.pf-v5-c-wizard__main {
    flex: 1 1 0
}
```
would work nicely for us.

Reply:
I also think it is a good idea to remove 
```
// Fixes min height issues on the main wizard container
.pf-v5-c-wizard__outer-wrap {
    min-height: 602px;
}
```, so let's keep that! :+1: 

We also might want to think about removing this too:
```
.pf-v5-c-wizard__nav {
    overflow-y: unset;
}
```

Otherwise, at small viewport heights, the nav bar won't be scrollable.

Reply:
work perfect!! thank you

Reply:
where did you find the  .pf-v5-c-wizard__main {
    flex: 1 1 0
} in patternFly? I looked here - https://www.patternfly.org/components/wizard